scripts/ci/run-ci-tests.sh

```diff
@@ -235,7 +235,7 @@ LAZY_OPTS="-p 2 -T $LAZY_TESTS $LAZY_EXCLUDE $ZDTM_OPTS"
 # shellcheck disable=SC2086
 ./test/zdtm.py run $LAZY_OPTS --remote-lazy-pages
 # shellcheck disable=SC2086
-./test/zdtm.py run $LAZY_OPTS --remote-lazy-pages --tls
+./test/zdtm.py run $LAZY_OPTS --remote-lazy-pages --tls -x lazy-thp
```

Comment:
pls add a fixme command that explains why we need to skip lazy-thp here

Reply:
Thank you Andrei, I added a "fixme" comment about the error.